.github/workflows/ci.yml

```diff
@@ -30,17 +30,44 @@ jobs:
             - name: Install dependencies
               run: composer install --prefer-dist --no-progress
 
+            - name: Run tests
+              run: composer test
+
+    php-cs-fixer:
```

Comment:
I need to split it because php-cs-fixer could not work with PHP 8.4 for now.
An anyway, it's useless to run it 3 times.

By extension, I also did that for phpstan